scripts/deploy-cloudflare.sh

```
@@ -0,0 +1,247 @@
+            *)
+                print_error "Unknown option: $1"
+                show_usage
+                exit 1
+                ;;
+        esac
+    done
+    
+    # Execute deployment steps
+    check_prerequisites
+    validate_environment
+    clean_build $([ "$clean_deps" = true ] && echo "--clean-deps")
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)